NwPluginAPI/Core/Extensions/ItemExtensions.cs

```diff
@@ -0,0 +1,80 @@
+namespace PluginAPI.Core
+{
+	/// <summary>
+	/// Item-related extensions.
+	/// </summary>
+	public static class ItemExtensions
+	{
+		/// <summary>
+		/// Check if an <see cref="T:ItemType">item</see> is an ammo.
+		/// </summary>
+		/// <param name="item">The item to be checked.</param>
+		/// <returns>Returns whether the <see cref="T:ItemType" /> is an ammo or not.</returns>
+		public static bool IsAmmo(this ItemType item)
+		{
+			return item is ItemType.Ammo9x19 or ItemType.Ammo12gauge or ItemType.Ammo44cal or ItemType.Ammo556x45 or ItemType.Ammo762x39;
+		}
+
+		/// <summary>
+		/// Check if an <see cref="T:ItemType">item</see> is a weapon.
+		/// </summary>
+		/// <param name="type">The item to be checked.</param>
+		/// <param name="checkMicro">Indicates whether the MicroHID item should be taken into account or not.</param>
```

Comment:
Just for clarity in the parameter name, maybe `includeMicro`and `includeJailbird` could be easier to understand without reading the description.